Game/Source/GDKTestGyms/Private/EventTracerComponent.cpp

```diff
@@ -32,13 +31,18 @@ void UEventTracerComponent::BeginPlay()
 void UEventTracerComponent::TimerFunction()
 {
 	// Create a user trace event for sending a property update
-	FUserSpanId SpanId = USpatialEventTracerUserInterface::TraceEvent(this, SpatialGDK::FSpatialTraceEventBuilder("user.send_component_property").GetEvent());
+	FUserSpanId SpanId = USpatialEventTracerUserInterface::TraceEvent(this, "user.send_component_property", "");
 	USpatialEventTracerUserInterface::TraceProperty(this, this, SpanId);
 
 	TestInt++;
 
 	// Create a user trace event for sending an RPC
-	SpanId = USpatialEventTracerUserInterface::TraceEvent(this, SpatialGDK::FSpatialTraceEventBuilder("user.send_rpc").GetEvent());
+	FEventTracerAddDataDelegate AddDataDelegate;
+	AddDataDelegate.BindLambda([](SpatialGDK::FSpatialTraceEventDataBuilder& Builder) {
+		Builder.AddKeyValue("Message", "Look mum, I'm adding data to a trace event using a delegate!");
```

Comment:
if this is testing code, lets be explicit about in a comment or similar, its not immediately obvious to me at least when looking at this code that its testing code.